src/ui-lib/sass/03-elements/_inline-text.scss

```diff
@@ -9,14 +9,15 @@ a[href] {
     color: $grav-co-interactive-secondary-hover;
   }
 
-  &:active {
-    background: $grav-co-interactive-secondary;
-    color: inherit;
-  }
-
   &:visited {
     color: $grav-co-interactive-secondary-visited;
   }
+
+  &:active {
+    background: $grav-co-interactive-secondary;
+    color: $grav-co-neutral-dirty-snow-white;
```

Comment:
Can you swap this for `$grav-co-interactive-secondary-contrast` please (it's defined in `_colors.scss`). It ultimately still ends up being Dirty Snow White, but we want to try to avoid directly referencing the named colours in our CSS.

The idea is that using intermediate variables, whose names express a purpose, mean we could reskin/rebrand a lot of our UI simply by fiddling with whatever named colours they map to. Otherwise, it becomes a tedious "find and replace" job.